apps/dashboard/src/components/workflow-editor/step-utils.ts

```diff
@@ -135,6 +140,16 @@ export const createStep = (type: StepTypeEnum, defaultLayoutId: string | undefin
     controlValue.layoutId = defaultLayoutId;
   }
 
+  if (type === StepTypeEnum.IN_APP) {
+    let path = INFO_AVATAR;
+    if (severity === SeverityLevelEnum.HIGH) {
+      path = ERROR_AVATAR;
+    } else if (severity === SeverityLevelEnum.MEDIUM) {
+      path = WARNING_AVATAR;
+    }
+    controlValue.avatar = `${window.location.origin}${path}`;
+  }
```

Comment:
whenever the `in-app` step is created, take into account the workflow severity level, and if it's set to high/medium, use the corresponding avatar icons